coverage.txt

```diff
@@ -0,0 +1,109 @@
+mode: set
```

Comment:
please rebase this PR editing this commit and removing this file - the rest looks good :+1:

Reply:
Sorted. Thanks